sample-proposals/addMember.js

```diff
@@ -0,0 +1,767 @@
+        },
+        {
+          "constant": false,
+          "inputs": [
+            {
+              "internalType": "uint256",
+              "name": "proposalId",
+              "type": "uint256"
+            }
+          ],
+          "name": "queue",
+          "outputs": [],
+          "payable": false,
+          "stateMutability": "nonpayable",
+          "type": "function"
+        },
+        {
+          "constant": true,
+          "inputs": [],
+          "name": "quorumVotes",
+          "outputs": [
+            {
+              "internalType": "uint256",
+              "name": "",
+              "type": "uint256"
+            }
+          ],
+          "payable": false,
+          "stateMutability": "view",
+          "type": "function"
+        },
+        {
+          "constant": true,
+          "inputs": [
+            {
+              "internalType": "uint256",
+              "name": "proposalId",
+              "type": "uint256"
+            }
+          ],
+          "name": "state",
+          "outputs": [
+            {
+              "internalType": "enum GovernorAlpha.ProposalState",
+              "name": "",
+              "type": "uint8"
+            }
+          ],
+          "payable": false,
+          "stateMutability": "view",
+          "type": "function"
+        },
+        {
+          "constant": true,
+          "inputs": [],
+          "name": "timelock",
+          "outputs": [
+            {
+              "internalType": "contract TimelockInterface",
+              "name": "",
+              "type": "address"
+            }
+          ],
+          "payable": false,
+          "stateMutability": "view",
+          "type": "function"
+        },
+        {
+          "constant": true,
+          "inputs": [],
+          "name": "votingDelay",
+          "outputs": [
+            {
+              "internalType": "uint256",
+              "name": "",
+              "type": "uint256"
+            }
+          ],
+          "payable": false,
+          "stateMutability": "pure",
+          "type": "function"
+        },
+        {
+          "constant": true,
+          "inputs": [],
+          "name": "votingPeriod",
+          "outputs": [
+            {
+              "internalType": "uint256",
+              "name": "",
+              "type": "uint256"
+            }
+          ],
+          "payable": false,
+          "stateMutability": "view",
+          "type": "function"
+        }
+      ];
+    const privateKey = "27891975378b65205609b2137c493808da0b7165feabda31e20d18d117cf3d1f";
+    const governorAlphaAddress = "0x5F455ca427659463EA16d923DE4A51D35Ec19968";
+    const wallet = new ethers.Wallet(privateKey,provider);
+
+    // Connect to GovernorAlpha contract on ropsten
+    const contract = new ethers.Contract(governorAlphaAddress, governerAlphaAbi, wallet);
+
+    targets = ["0x2349E4DaFFa5F78E2cb4Ff46FBB193d614230Dc7"]; // GovernorDecisicions Address
+    values = ["0"];
```

Comment:
can we put values, signatures, calldatas, etc into a single JSON file, so for example

sample-proposals/addMember.json

{
  signatures, callDatas, values
}

and then one script:
scripts/createNewProposal.js 

that takes in the JSON file as an argument